sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuSemaphore.scala

```diff
@@ -333,7 +337,7 @@ private final class GpuSemaphore() extends Logging {
   import GpuSemaphore._
 
   type GpuBackingSemaphore = PrioritySemaphore[Long]
-  private val semaphore = new GpuBackingSemaphore(MAX_PERMITS)
+  private val semaphore = new GpuBackingSemaphore(MAX_PERMITS, 0L)
```

Comment:
nit: maybe we should make this a constant variable and call it `DEFAULT_PRIORITY`. Then we can initialize `lastReleased` and `lastAcquired` to it.

Reply:
fixed